events/lastAvatarStandingTrivia/entityScripts/triviaColorCheckYellow.js

```diff
@@ -10,15 +10,15 @@
 
```

Comment:
seems like this file (and the other `triviaColorCheck*.js` files) could be refactored into one file to DRY up the code and reduce complexity. don't do this now; it'll take too much time. but for the future!!